egov/egov-commons/src/main/java/org/egov/eis/entity/DrawingOfficer.java

```diff
@@ -129,6 +131,14 @@ public void setAccountNumber(final String accountNumber) {
         this.accountNumber = accountNumber;
     }
 
+    public String getIfscCode() {
```

Comment:
Don't need to have getter and setter for ifsc code if already the setters and getters are overridden. Remove these and keep the overridden getter and setter